downstream/modules/aap-on-azure/proc-azure-update-route-tables.adoc

```diff
@@ -9,13 +9,13 @@ Before you update the route tables, confirm that you satisfy the xref:proc-azure
 
 . Navigate to *Route Tables* in the Azure portal.
 . As part of your hub-and-spoke configuration, you created one or more route tables to define the routes between the networks. Click on one of these route tables.
-. From the route table menu bar, click *Routes* > *Add*.
+. From the route table menu bar, click btn:[Routes] > btn:[Add].
```

Comment:
This is a menu selection and I would rewrite to:

From the route table menu bar, select menu:Routes[Add].